client/dashboard/app/routes/applications.technical-project.tsx

```diff
@@ -35,9 +36,16 @@ export const meta: V2_MetaFunction = () => {
 
 export const links = () => [...projectTilesLinks(), ...projectLinks()];
 
-export const loader: LoaderFunction = ({ request }) => {
+export const loader: LoaderFunction = async ({ request }) => {
   const url = new URL(request.url);
   const slug = url.searchParams.get("slug");
+  const userDetails = await getUserDetails(request);
+  if (userDetails.application && userDetails.application.length > 0) {
+    const applicationExists = userDetails.application[0];
+    if (!(applicationExists.status === "draft")) {
```

Comment:
also, i dont think its a good idea to use "draft" everywhere. create enum or create an object with as const In a shared folder and use it everywhere for things like pending, and other constant things.